requirements.txt

```diff
@@ -26,211 +26,226 @@
 -r requirements.pinned.txt
 
 # Requirements not on PyPI
--e git+https://github.com/lnielsen/invenio-migrator.git@#egg=invenio-migrator
 -e git+https://github.com/zenodo/zenodo-migrator.git#egg=zenodo-migrator
 
 # PyPI requirements
-alabaster==0.7.9
-alembic==0.8.8
-amqp==1.4.9
-anyjson==0.3.3
+alabaster==0.7.10
+alembic==0.9.2
+amqp==2.1.4
+angular-gettext-babel==0.3
 apipkg==1.4
-arrow==0.8.0
-Babel==2.3.4
+arrow==0.10.0
+asn1crypto==0.22.0
+autosemver==0.5.2
+Babel==2.4.0
 backports.shutil-get-terminal-size==1.0.0
-billiard==3.3.0.23
-binaryornot==0.4.0
-bleach==1.5.0
+billiard==3.5.0.2
+binaryornot==0.4.3
+bleach==2.0.0
 blinker==1.4
-cairocffi==0.7.2
+cairocffi==0.8.0
 CairoSVG==1.0.22
-cchardet==1.1.1
-celery==3.1.25
-cffi==1.9.1
-chardet==2.3.0
-check-manifest==0.34
-citeproc-py==0.3.0
+cchardet==2.1.1
+celery==4.0.2
+certifi==2017.4.17
+cffi==1.10.0
+chardet==3.0.4
+check-manifest==0.35
+citeproc-py==0.4.0
 citeproc-py-styles==0.1.0
-click==6.6
+click==6.7
 configparser==3.5.0
-cookiecutter==1.4.0
-coverage==4.2
-cryptography==1.5.3
+cookiecutter==1.5.1
+coverage==4.4.1
+cryptography==1.9
 datacite==0.3.0
 dcxml==0.1.0
-decorator==4.0.10
-dictdiffer==0.6.0
-docutils==0.12
-dojson==1.2.1
-elasticsearch==2.4.0
+decorator==4.0.11
+dictdiffer==0.6.1
+docutils==0.13.1
+dojson==1.3.1
+dulwich==0.17.3
+elasticsearch==2.4.1
 elasticsearch-dsl==2.2.0
-entrypoints==0.2.2
+entrypoints==0.2.3
+enum34==1.1.6
 execnet==1.4.1
-Flask==0.11.1
-Flask-Admin==1.4.2
+Flask==0.12.2
+Flask-Admin==1.5.0
 Flask-Alembic==2.0.1
 Flask-Assets==0.12
 Flask-BabelEx==0.9.3
 Flask-Breadcrumbs==0.4.0
 Flask-Cache==0.13.1
-Flask-CeleryExt==0.2.2
+Flask-CeleryExt==0.3.0
 Flask-Collect==1.2.2
-Flask-Cors==3.0.2
-Flask-DebugToolbar==0.10.0
-Flask-Konch==1.1.0
+Flask-Cors==3.0.3
+Flask-DebugToolbar==0.10.1
+flask-konch==1.1.0
 Flask-KVSession==0.6.2
-Flask-Limiter==0.9.3
 Flask-Login==0.4.0
 Flask-Mail==0.9.1
 Flask-Menu==0.5.1
-Flask-OAuthlib==0.9.3
+Flask-OAuthlib==0.9.4
 Flask-Principal==0.4.0
-Flask-Security-Fork==1.8.1
-Flask-SQLAlchemy==2.1
-Flask-WTF==0.13.1
+Flask-Security-Fork==2.0.1
+Flask-SQLAlchemy==2.2
+Flask-WTF==0.14.2
```

Comment:
Check this one before deploying because it has caused us some [headaches](https://github.com/inveniosoftware/invenio-oauthclient/issues/115) with Invenio-OAuthClient and the CERN app, I don't know if this is also the case for GitHub.